### PR TITLE
Form::States:: Add filter and make form data dynamic with child to parent passing with Lifting State Up

### PR DESCRIPTION
**Pull request by abdulhanan001**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import './App.css';
 import Expenses from './components/Expenses/Expenses';
 import NewExpense from './components/NewExpense/NewExpense';
+import ExpenseDate from './components/Expenses/ExpenseDate';
```
Comment (abdulhanan001) [inline src/App.js:4]: remove unnecessary line

**Review by abdulhanan001 — COMMENTED** [review]
changes

File: src/components/Expenses/Expenses.js [hunk 2]
```diff
@@ -1,18 +1,44 @@
-import React from 'react'; // add React only for follow convention for jsx
-import ExpenseItem from './ExpenseItem';
-
-const Expenses = (props)  => {
+import React from "react"; // add React only for follow convention for jsx
+import ExpenseItem from "./ExpenseItem";
+import { useState } from "react";
+import ExpensesFilter from "./ExpensesFilter";
 
+const Expenses = (props) => {
```
Comment (mianazmatali9) [inline src/components/Expenses/Expenses.js:6]:
```suggestion
const Expenses = ({ expenses }) => {
```

File: src/components/Expenses/ExpensesFilter.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './ExpensesFilter.css';
+
+const ExpensesFilter = (props) => {
+  const dropdownChangeHandler = (event) => {
+    props.onChangeFilter(event.target.value)
+  }
```
Comment (mianazmatali9) [inline src/components/Expenses/ExpensesFilter.js:7]: Configure Eslint.

**Pull request merged**